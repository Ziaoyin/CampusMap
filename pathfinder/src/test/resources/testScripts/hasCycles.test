# tests that cycles don't impact shortest path
CreateGraph graph
AddNode graph n1
AddNode graph n2
AddNode graph n3

AddEdge graph n1 n2 3
AddEdge graph n2 n1 5
AddEdge graph n2 n3 1

FindPath graph n1 n3