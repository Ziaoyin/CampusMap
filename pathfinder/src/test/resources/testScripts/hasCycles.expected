# tests that cycles don't impact shortest path
created graph graph
added node n1 to graph
added node n2 to graph
added node n3 to graph

added edge 3.000 from n1 to n2 in graph
added edge 5.000 from n2 to n1 in graph
added edge 1.000 from n2 to n3 in graph

path from n1 to n3:
n1 to n2 with weight 3.000
n2 to n3 with weight 1.000
total cost: 4.000